impf/alert.py

```diff
@@ -68,13 +69,24 @@ def send_alert(message: str) -> None:
         telegram_send(message)
     if settings.PUSHOVER_ENABLED:
         pushover_send(message)
-
+    if settings.SYNOLOGYCHAT_ENABLED:
+        synologychat_send(message)
 
 @alert_resilience
 def execute_command() -> None:
     os.system(get_command())
 
 
+@alert_resilience
+def synologychat_send(message: str) -> None:
+    payload = 'payload=' + json.dumps({'text': message})
+    response = requests.post(settings.SYNOLOGYCHAT_WEBHOOK_URL, payload, verify=False)
+    response_json = response.json()
+    success = str(response_json["success"])
+    if success == 'False':
```

Comment:
We can do it like that, but it's usually best to set dicts to `response_json.get('success')` to avoid an IndexError in case the variable is not set for whatever reason. :)